src/components/Calculator.js

```diff
@@ -1,18 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
+import calculate from '../logic/calculate'; // eslint-disable-line
 
-class Calculator extends React.Component { // eslint-disable-line
-  render() {
-    const figures = [{ val: 'AC' }, { val: '+/-' }, { val: '%' }, { val: '/' }, { val: 7 }, { val: 8 }, { val: 9 }, { val: 'x' }, { val: 4 }, { val: 5 }, { val: 6 }, { val: '-' }, { val: 1 }, { val: 2 }, { val: 3 }, { val: '+' }, { val: 0 }, { val: '.' }, { val: '=' }];
-    const id = 'item';
-    return (
-      <div className="container">
-        <div className="display">0</div>
-        {figures.map((fig) => (
-          <button className="press" type="button" key="fig.val" id={id + figures.indexOf(fig)}>{fig.val}</button>
-        ))}
+const Calculator = () => {
+  const [state, setState] = useState({ total: null, operation: null, next: null });
+
+  const figures = [{ val: 'AC' }, { val: '+/-' }, { val: '%' }, { val: '/' }, { val: 7 }, { val: 8 }, { val: 9 }, { val: 'x' }, { val: 4 }, { val: 5 }, { val: 6 }, { val: '-' }, { val: 1 }, { val: 2 }, { val: 3 }, { val: '+' }, { val: 0 }, { val: '.' }, { val: '=' }];
+  const id = 'item';
```

Comment:
- [ ] The division sign is not working at the moment because you are using the `/` symbol. The sign used in your `operate.js` file for division is not `/` but `÷`. Kindly edit one so it works. It did on my end when I changed the sign in the `operate.js` file to `/`. Please work on that 👍